databricks_sql_connector/pyproject.toml

```diff
@@ -0,0 +1,33 @@
+[tool.poetry]
+name = "databricks-sql-connector"
+version = "1.0.0"
+description = "Databricks SQL Connector for Python"
+authors = ["Databricks <databricks-sql-connector-maintainers@databricks.com>"]
+license = "Apache-2.0"
+readme = "README.md"
+include = ["CHANGELOG.md"]
+
+[tool.poetry.dependencies]
+python = "^3.8.0"
+pyarrow = ">=14.0.1,<17"
+databricks_sql_connector_core = { version = ">=1.0.0"}
```

Comment:
This package should basically have no other dependencies except `databricks_sql_connector_core`. Installation of optional deps should be achieved by adding `extras=["all"]` option (like - `databricks_sql_connector_core = { version = ">=1.0.0", extras=["all"]}`. Have you tried it? If there are any caveats with this approach - please share your thoughts